hw/top_earlgrey/data/ip/chip_sysrst_ctrl_testplan.hjson

```diff
@@ -5,77 +5,191 @@
   name: chip_sysrst_ctrl
   testpoints: [
     {
-      name: chip_sw_sysrst_ctrl_combo_reset
-      desc: '''Verify that sysrst_ctrl issue reset on combo detection
+      name: chip_sw_sysrst_ctrl_reset
+      desc: '''Verify the SYSRST ctrl can reset the chip from normal state.
 
-            - Initialize sysrst_ctrl with combo detect and output of combo detect as rst_req assertion.
-            - Wait for a delay greater than combo detect timer
-            - Check if there is a reset and if reset source is sysrst_ctrl
-            - Check if sysrst_ctrl.WKUP_STATUS.wakeup_sts is set
+            - Read the reset cause register in rstmgr to confirm we are in POR reset phase.
+            - Program one of the com_sel_ctl_* CSRs to choose a set of inputs to be detected as
+              the chip reset signal.
+            - Program the associated detection timer.
+            - Program the detection outcome CSR's (com_out_ctl) chip reset bit to 1.
+            - After some time, set the SYSRST ctrl inputs at the chip IOs to the programmed
+              combination for the duration of the detection timer.
+            - The pwrmgr will power cycle the chip once it receives the chip reset input.
+            - Check that ec_rst_l and flash_wp_l (on pads IOR8 and IOR9) are asserted right after
+              the pwrmgr has power cycled the system.
+            - Read the reset cause register after boot up to confirm peripheral reset phase.
+            - Read the pwrmgr reset status register to confirm chip reset.
+            - Read the com_sel_ctl_* CSR in SYSRST ctrl we programmed earlier - it should have been
+              reset.
             '''
       features: ["SYSRST_CTRL.COMBO_DETECT"]
-      stage: V3
+      stage: V2
       si_stage: SV3
       lc_states: ["DEV", "PROD", "PROD_END", "RMA"]
       tests: ["chip_sw_pwrmgr_sysrst_ctrl_reset", "chip_sw_sysrst_ctrl_reset"]
     }
     {
-      name: chip_sw_sysrst_ctrl_input
-      desc: '''Verify that SW can access the raw value of input signal via sysrst_ctrl
-
-            - Initialize sysrst_ctrl
-            - set the following input signals with a random value
-              - pwrb_in_i
-              - key2_in_i
-              - key1_in_i
-              - key0_in_i
-              - ac_present_i
-              - ec_rst_l_i
-              - flash_wp_l_i
```

Comment:
I think this stuff was probably part of the existing DV test. It's reasonable to decide we don't need to do this for sival but, in that case, please could you split this into two entries (one for DV / one for sival)?

Reply:
The description of the test has not change, the text is just formatted differently: if you look at the original testpoint is the chip testplan (which is the one I kept), it says:
```Drive a known value on ac_reset, ec_rst_l, flash_wp_l, pwrb, lid_open and key* pins at
              the chip inputs.```
The sival test (already implementeD) does the same thing as the DV test and exercises all pins.

Reply:
Duh, you're completely right. I hadn't tracked which filename was which! Looks sensible to me.